apps/institution-service/src/tasks/loadInstitutionsFromJson.ts

```diff
@@ -96,14 +101,29 @@ async function loadInstitutionData() {
         isActive: false,
         aggregatorId: finicityAggregator?.id,
         aggregator_institution_id: finicity.id,
-        institution_id: institution.ucp_id,
+        institution_id: institution.id,
         supports_oauth: finicity.supports_oauth,
         supports_identification: finicity.supports_identification,
         supports_verification: finicity.supports_verification,
         supports_history: finicity.supports_history,
         supports_aggregation: finicity.supports_aggregation,
       } as AggregatorIntegration);
     }
+
+    if (institution?.testExampleA?.id) {
```

Comment:
Don't we need testExampleB?

Reply:
oh yeah is that the only other one

Reply:
I think I've added all the necessary test institutions to the json file now

Reply:
Yeah testExampleB is the only other one.